library/Icinga/Chart/Donut.php

```diff
@@ -344,7 +344,7 @@ protected function encode($content)
 
     protected function renderAttributes(array $attributes)
     {
-        $html = [];
```

Comment:
@theFeu I'm afraid we can't code like this as long as we don't have green light from @lippserd.